PelotonEppSdk/Models/VariableType.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace PelotonEppSdk.Models
+{
+    public enum VariableType
```

Comment:
I gather that the VariableType enum is meant to be used with EventCustomField, but it is never used. Can you demonstrate its usage, perhaps with a unit test?